tests/pytorch/nightly/resnet50-mp.libsonnet

```diff
@@ -75,7 +75,7 @@ local tpus = import "templates/tpus.libsonnet";
     accelerator: tpus.v3_8,
   },
   configs: [
-    resnet50_MP + v3_8 + convergence + timeouts.Hours(26),
+    resnet50_MP + v3_8 + convergence + timeouts.Hours(26) + mixins.PreemptibleTpu,
```

Comment:
I don't see this changed .yaml file in this PR. Using preemptible here could make sense to collect data but I feel like such a long running test is going to get preempted a lot

Reply:
This test was already using preemptible. See the change on line 58.